src/timepicker/styles/timePicker.m.css

```diff
@@ -0,0 +1,77 @@
+@import '../../common/styles/variables.css';
+
+.root,
+.root * {
+	box-sizing: border-box;
+}
+
+.root {
+	display: inline-block;
+	position: relative;
+	width: var(--form-field-width);
```

Comment:
We probably need to revisit this when we get an actual theme. Currently all input-based elements use `--form-field-width`.